### ISSUE TITLE
Reduce height of title box

### ISSUE THREAD

**Issue by jmburgos** [key-amb/hugo-theme-bootie-docs#24]
Hi Kiyoshi,
Sorry for the newbie question, but how do you reduce the height of the gray box that surrounds the site title and description?  I would like to have less empty space.
Many thanks!

**Comment by ethanmad** [comment]
This is what I have in my site, but I've modified the jumbotron quite a bit. (See http://umich.edu/~fencing and UM-Fencing/club-theme.) Try it; I think it should work.

[`bootie-docs.css`](https://github.com/UM-Fencing/club-theme/blob/a52b93a44d3ba3462c86a5ab7c50ae5c2a2eb51d/static/css/bootie-docs.css#L55):
```css
.jumbotron {
    height: 420px;
}
```

**Comment by key-amb** [comment]
Thank you, @ethanmad 
I'll take care of this later.

**Comment by key-amb** [comment]
Hello @jmburgos cc: @ethanmad 

I just implemented an feature to customize site's CSS: #27 
I recommend you to update bootie-docs theme.
Note about the change in #26 , too.

Then, you can add your custom CSS as `static/css/site.css`.

And, to customize [Jumbotron of Bootstap](http://getbootstrap.com/components/#jumbotron), refer to the css codes around here: https://github.com/twbs/bootstrap/blob/v3.3.6/dist/css/bootstrap.css#L4961-L5003
(We currently use v3.3.6 for bootie-docs)

Finally, I think adding `static/css/site.css` with following lines will meet your request:

```css
/* just an example codes */
.jumbotron {
  padding-top: 10px;
  padding-bottom: 10px;
  margin-bottom: 10px;
}
```

**Comment by ethanmad** [comment]
Great, thank you, @key-amb. Now merging in your changes should be easier for my and other forks.
